core/templates/dev/head/pages/exploration_editor/settings_tab/SettingsTab.js

```diff
@@ -54,6 +54,7 @@ oppia.controller('SettingsTab', [
     }
 
     $scope.isRolesFormOpen = false;
+    $scope.titleAdd = false;
```

Comment:
This naming feels odd. As the variable is a boolean, consider naming it to something like "hasTitleBeenAdded".